Services/Documents/DocumentService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.Documents;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+using System.IO;
+
+namespace Andculture.Sitefinity.Testing.Services.Documents
+{
+    public class DocumentService : ODataService<Document>
+    {
+        public DocumentService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/documents";
+        }
+
+        public IRestResponse Upload(Document model)
+        {
+            string documentTitle = "digital_marketing";
+            string documentExtension = ".pdf";
+            string docFullName = string.Concat(documentTitle, documentExtension);
```

Comment:
Suggestion: Do away with documentTitle and documentExtension since you're only ever concatenating them both (once on line: 22 and once on line 23. I would define a variable that includes the full name:
```
string documentName = "digital_marketing.pdf"
```

Then when building the file path looks like:
```
string filePath = string.Concat(Directory.GetCurrentDirectory(), @"\Media\", documentName);
```